tests/testUtils.ts

```diff
@@ -371,4 +371,520 @@ export const createTickArray = (size: number) => {
   return ticks
 }
 
-//export const dataApy: ApyPoolParams = {}
+export const jsonArrayToTicks = (data: any[]) => {
+  const ticks: Tick[] = []
+
+  data.forEach(tick => {
+    ticks.push({
+      index: tick.index,
+      sign: tick.sign,
+      bump: tick.bump,
+      liquidityChange: { v: new BN(tick.liquidityChange.v) },
+      liquidityGross: { v: new BN(tick.liquidityGross.v) },
+      sqrtPrice: { v: new BN(tick.sqrtPrice.v) },
+      feeGrowthOutsideX: { v: new BN(tick.feeGrowthOutsideX.v) },
+      feeGrowthOutsideY: { v: new BN(tick.feeGrowthOutsideY.v) },
+      secondsPerLiquidityOutside: {
+        v: new BN(tick.secondsPerLiquidityOutside.v)
+      },
+      pool: new PublicKey(tick.pool)
+    })
+  })
+
+  return ticks
+}
+
+export const usdcUsdhPool = {
+  feeTier: { fee: '100000000' },
+  volumeX: 2948000000,
+  volumeY: 2552000000,
+  ticksPreviousSnapshot: [
+    {
+      index: -13860,
+      sign: true,
+      bump: 251,
+      liquidityChange: { v: '22964554844140308' },
+      liquidityGross: { v: '22964554844140308' },
+      sqrtPrice: { v: '500090922499000000000000' },
+      feeGrowthOutsideX: { v: '124251168448391763' },
+      feeGrowthOutsideY: { v: '146254149906104013' },
+      secondsPerLiquidityOutside: { v: '264697' },
+      pool: 'FwiuNR91xfiUvWiBu4gieK4SFmh9qjMhYS9ebyYJ8PGj'
```

Comment:
pool filed seems to be redundant. Just mock data, so it isn't problem

Reply:
this field has to be there because it's tick interface